### ISSUE TITLE
Add Features to Editor after Loading

### ISSUE THREAD

**Issue by zoenberger** [uber/nebula.gl#294]
This may be a my lack of understanding how to use this  module, but I want to load Polygon data from my backend on load. After it's loaded, I want to add those features using the Editor.addFeatures([features]) method. This is _mostly_ working for me, but it's unreliable and hacky due to it being unclear the proper way to reference the Editor object.

The issue is that often when I call the addFeatures method, the Editor reference is not assigned. 
Note: I followed Editor reference example from here: [https://nebula.gl/docs/api-reference/react-map-gl-draw/react-map-gl-draw](https://nebula.gl/docs/api-reference/react-map-gl-draw/react-map-gl-draw)

I've uploaded a repo (HERE: [https://github.com/zoenberger/React-Map-Draw-GL---Polygon-Test](https://github.com/zoenberger/React-Map-Draw-GL---Polygon-Test) ) for you to check out exactly what I'm trying to do. 

`  useEffect(() => {

    setMarkers(testVehicle.map((v, i) => _renderMarker(v, i)));

    // The button  on page will call this method: addZone();

    if (editorRef.addFeatures) {
      editorRef.addFeatures(testZone);
    } else {
      setTimeout(() => {
        console.log("After three seconds, attempt to add test zone again...");
        if (editorRef.addFeatures) {
          editorRef.addFeatures(testZone);
        }
      }, 3000);
    }
  }, []);`

**Comment by intelcoder** [comment]
Can you modify feature by dragging vertex ?

**Comment by zoenberger** [comment]
Per the documentation, you can either supply an array of features to the Editor component it allow the Editor component to create an array to manage features internally.

I tried supplying my own feature array, and when I did that, I could NOT modify a feature (separate issue than described on this post).

Using the code repo I provided, YES you can modify features.

**Comment by intelcoder** [comment]
I just wanted to make sure user provided features are not editable at the moment. I created new issue with that matter. Thanks

**Comment by xintongxia** [comment]
It is a bit late to respond. But this is fixed in v0.18.3. But it is related to this issue.

A workaround is calling addFeatures in onLoad callback of the map.

```
    <MapGL
        ...
        onLoad={() => {
          // note: there is an issue with react-map-gl https://github.com/visgl/react-map-gl/issues/1098
          // Editor isn't available in `componentDidMount`
          // A workaround is calling` addFeatures`  when `map` loaded.
          this._editorRef.addFeatures(features);
        }}
      >
        <Editor
          ...
          ref={(_) => (this._editorRef = _)}
        />
      </MapGL>

```
